### ISSUE TITLE
Getting started page should also state post installation steps (as well as "Hello early user," app start page)

### ISSUE THREAD

**Issue by tmotyl** [ActivityWatch/docs#92]
getting started page should mention what to do after installation e.g. installing browser extensions or editor plugins.
Maybe there is a way to offer installation of the browser plugin after visiting the Activity Watcher?

**Comment by ErikBjare** [comment]
Since the web extension isn't really necessary, I've hidden this under the 'Activity -> Browser' tab/visualizations:

![image](https://user-images.githubusercontent.com/1405370/187732388-5e0ea7f4-d24a-4c1c-9ba9-f7bb7dc6818d.png)

Perhaps it's good UX to get people to do it straight away though. There is another benefit to having more people install the extension: improve our estimates of active users (which we only get from web extension installations: https://activitywatch.net/stats/)

Could detect browser and link directly to extension store pages (most users), and also link to watcher page in the docs (for more advanced users).

**Comment by tmotyl** [comment]
yes, the more advanced/determined will find it.
However, what block  AW adoption is the high entry barrier.

Ideally, user after first visit of the app, should be offered a guided tour over the AW in my opinion, but that's another story.

**Comment by tmotyl** [comment]
yes, showing a google chrome store badge (https://developer.chrome.com/docs/webstore/branding/#using-the-chrome-web-store-badge) with link to google chrome store is the way to go
